hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/records/RouterMasterKey.java

```diff
@@ -130,4 +130,10 @@ public boolean equals(Object right) {
         .append(getKeyBytes().array(), r.getKeyBytes())
         .isEquals();
   }
+
+  @Override
+  public String toString() {
+    return "RouterMasterKey [keyId = " + getKeyId()
```

Comment:
StringBuilder might be better for this.

Reply:
Thank you very much for your help to review the code, I will use `StringBuilder` to refactor this part of the code.